packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -123,7 +136,7 @@ const Summary: React.FunctionComponent< Props > = ( { siteId } ) => {
 				<div className="focused-launch-summary__data-input">
 					<div className="focused-launch-summary__section">
 						<label className="focused-launch-summary__label">
-							{ __( '3. Confirm your plan', __i18n_text_domain__ ) }
+							{ stepNumber++ }. { __( 'Confirm your plan', __i18n_text_domain__ ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.